src/Shared/Controls/Settings/MusicSources/ConnectNew/Ipfs/LandingPage.xaml

```diff
@@ -0,0 +1,73 @@
+﻿<Page
+    x:Class="StrixMusic.Controls.Settings.MusicSources.ConnectNew.Ipfs.LandingPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+    xmlns:local="using:StrixMusic.Controls.Settings.MusicSources.ConnectNew.Ipfs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+
+    <StackPanel
+        MaxWidth="550"
+        Margin="20"
+        Spacing="15">
+
+        <labs:SettingsExpander
+            Description="Provide the root directory Cid."
+            Header="Configure IPFS folder"
+            IsEnabled="{x:Bind IsNotNull(Settings), Mode=OneWay}"
+            IsExpanded="True"
+            Visibility="{x:Bind NullToVisibility(_param.AppRoot.Ipfs.Client)}">
```

Comment:
We shouldn't bind to fields. Add an `[ObservableProperty]` to the field and use the generated property instead.